Realm/Tests/RealmTests.mm

```diff
@@ -175,6 +177,58 @@ - (void)testFileFormatUpgradeRequiredButDisabled {
     RLMAssertThrowsWithCodeMatching([RLMRealm realmWithConfiguration:config error:nil], RLMErrorFileFormatUpgradeRequired);
 }
 
+- (void)testExceedingVirtualAddressSpace {
```

Comment:
How long does this test run for on an OS X machine with a lot of memory? I guess it's relatively quick since it maps in large chunks (1GB), and doesn't touch any of the mapped pages?